app/Rom.php

```diff
@@ -267,10 +267,14 @@ public function setStartingEquipment(ItemCollection $items) {
 					break;
 				case 'SilverArrowUpgrade':
 					$equipment[0x38E] |= 0b01000000;
+					if ($this->config('rom.rupeeBow', true))
```

Comment:
rupee bow should be defaulted to false, also missing curly braces

Reply:
Changed to
```
if ($this->config('rom.rupeeBow', false)) {
	...
}
```